.php_cs

```diff
@@ -1,29 +1,28 @@
 #!/usr/bin/env php
 <?php
 
-$finder = Symfony\CS\Finder::create()
-    ->ignoreDotFiles(true)
-    ->ignoreVCS(true)
-    ->exclude('app/config')
-    ->exclude('app/data')
-    ->exclude('app/Resources')
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
     ->exclude('var')
     ->exclude('vendor')
-    ->exclude('web/bundles')
-    ->exclude('web/css')
-    ->exclude('web/fonts')
-    ->exclude('web/js')
     ->notPath('web/config.php')
-    ->in(__DIR__)
+    // Doctrine fixtures are excluded to avoid these PHP-CS-Fixer errors:
+    // "Files were not fixed due to errors reported during linting before fixing"
```

Comment:
IMO, we must indeed rather fix the issue, which is a bad git conflict resolution: https://github.com/symfony/symfony-demo/blob/eba7c9f21bbd2dca312c6f767b77aafc60d47d33/src/AppBundle/DataFixtures/ORM/PostFixtures.php#L80